src/engine/game/gui/ingame/ingame.cpp

```diff
@@ -80,7 +80,7 @@ void InGameMenu::load() {
 
 void InGameMenu::loadEquipment() {
     _equip = make_unique<Equipment>(_game);
-    _equip->load();
+    _equip->load(this);
```

Comment:
`InGameMenu` could be injected as a reference in the `Equipment` ctor. Not that it will be different across multiple `load` calls.